backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/VmCommand.java

```diff
@@ -624,17 +624,13 @@ public boolean isSupportedByManagedBlockStorageDomain(StorageDomain storageDomai
     }
 
     public boolean isVmDuringBackup() {
-        if (!shouldBlockDuringBackup()) {
-            return false;
```

Comment:
to be more specific, what I meant here is to deprecate the 'use_active' option in the api and then to use the config value here to save calls to the database

Reply:
I'm reluctant on closing the door on support per-request backup strategy selection since we already support it anyway, and since we can only have one running backup per VM and finished backups are cleared after 15 minutes the call should be relatively cheap 

(also, I think we may have a scenario where we have two backup types on the same VM after upgrade, but I'm not sure if it's actually possible :)

Reply:
> I'm reluctant on closing the door on support per-request backup strategy selection since we already support it anyway, 

the sooner we deprecate it the easier it would be to ignore it ;)